model/installation.go

```diff
@@ -81,6 +85,102 @@ type InstallationFilter struct {
 	Name            string
 }
 
+type AllowedIPRanges []AllowedIPRange
+
+type AllowedIPRange struct {
+	CIDRBlock   string
+	Description string
+	Enabled     bool
+	// TODO - necessary?
```

Comment:
@nickmisasi leftover?

Reply:
We still haven't decided :) I think it would be useful to store the MM User ID